charts/lws/templates/manager/configmap.yaml

```diff
@@ -0,0 +1,32 @@
+{{- /*
```

Comment:
Seems we don't require for this, let's keep consistent.

Reply:
Thank you for review. Removed.